librarian_server/stores/pathinfo.py

```diff
@@ -19,8 +19,8 @@ class PathInfo(BaseModel):
     "Path being considered"
     filetype: str
     "File type at the path (e.g. png)"
-    md5: str
-    "MD5 sum of the file at the path"
+    checksum: str
```

Comment:
Does this mean that we need to update the database info to reflect a column name change? It doesn't look like there is an alembic upgrade as part of this PR, so I'm guess not, so I suppose I'm just wondering how pydantic squares this change with previous database entries.

Reply:
No, PathInfo is just an object that is created in response to a path information request. It is never stored in the database.